.travis.yml

```diff
@@ -42,6 +42,7 @@ before_install:
   - export BUILD_DIRECTORY="$TRAVIS_BUILD_DIR"
   - if [[ $TRAVIS_OS_NAME == "osx" ]]; then
         export OS_NAME="macos";
+        export COMPILER="gcc"
     else
         export OS_NAME="linux";
```

Comment:
I think that we should remove "global" `COMPILER` (L14) and add here `export COMPILER="clang";`, since we use different compilers for different OS. At present it's seems to be a little bit confusing.